README.md

```diff
@@ -265,7 +265,7 @@ The PSPDFKit React Native iOS Wrapper allows you to specify a custom grouping fo
 - Android Build Tools 28.0.3 (PSPDFKit module)
 - Android Gradle plugin >= 3.2.1
 - PSPDFKit >= 5.0.1
-- react-native >= 0.57.8
+- react-native >= 0.59.2
```

Comment:
The wrapper works with this version but our Catalog Example doesn't, might be worth mentioning.

Reply:
I'm updating the Catalog too if we don't have blockers

Reply:
There's already a PR for iOS, I think you should add the needed changes there.
The only thing I noticed when I tried were dependency conflicts with react-native-camera, but you can apparently override the used dependencies in your build-script so that should fix it.
https://github.com/react-native-community/react-native-camera#android